### PR TITLE
optionally enable mdns service in firewalld

### PR DESCRIPTION
**Pull request by zshi-redhat**
Related-Issue: https://issues.redhat.com/browse/OCPBUGS-6862

Signed-off-by: Zenghui Shi <zshi@redhat.com>

### CODE REVIEW

File: docs/howto_firewall.md [hunk 1]
```diff
@@ -41,5 +40,6 @@ sudo firewall-cmd --permanent --zone=public --add-port=5353/udp
 sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/tcp
 sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/udp
 sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp
+sudo firewall-cmd --permanent --add-service=mdns
```
Comment (fzdarsky) [inline docs/howto_firewall.md:43]: Should the zone be explicit here?

Comment (zshi-redhat) [reply docs/howto_firewall.md]: added public zone.

**Comment by ggiguash** [comment]
@zshi-redhat, please add a JIRA issue for this PR and unhold it to merge.
/hold
/lgtm

**Pull request merged**